kobuki_safety_controller/package.xml

```diff
@@ -17,27 +17,24 @@
   <url type="repository">https://github.com/kobuki-base/kobuki_ros</url>
   <url type="website">http://ros.org/wiki/kobuki_safety_controller</url>
```

Comment:
<url type="website">https://index.ros.org/p/kobuki_safety_controller/github-kobuki-base-kobuki_ros</url>

Reply:
Fixed in 9b22898.